odoo/modules/module.py

```diff
@@ -153,6 +154,21 @@ def initialize_sys_path():
         sys.meta_path.insert(0, AddonsHook())
         hooked = True
 
+
+def is_addons_path(path, recursive=True):
+    for path, dirnames, _f in os.walk(path):
+        for f in dirnames:
+            modpath = os.path.join(path, f)
+            if os.path.isdir(modpath):
+                def hasfile(filename):
+                    return os.path.isfile(os.path.join(modpath, filename))
+                if hasfile('__init__.py') and any(hasfile(mname) for mname in MANIFEST_NAMES):
```

Comment:
Not sure about that one for support of data-only addon, but I don't really know.

Reply:
This is a simple move of the method from one file to another.
See
https://github.com/odoo/odoo/pull/27217/files#diff-3813bc998e290bd756b27aef89662e82L495